rustfst/src/algorithms/tr_unique.rs

```diff
@@ -89,4 +95,36 @@ mod test {
 
         Ok(())
     }
+
+    //#[test]
+    //fn test_tr_map_unique_1() -> Result<()> {
```

Comment:
commented test

Reply:
Same as the python test, commented because it won't pass without the fix